docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,228 @@
+"""
+=====================
+Domino Physics Simulation
+=====================
+
+This example simulation shows how to use pybullet to render physics simulations
+in fury. In this example we specifically render a series of Dominoes which are
+under Domino Effect.
+
+"""
+###############################################################################
```

Comment:
I don't think this line is necessary :thinking: . As the doc-string itself will be considered as a different section.